adapter/main.go

```diff
@@ -139,7 +142,19 @@ func (a *Adapter) makeProvider(ctx context.Context, globalHTTPTimeout time.Durat
 
 	broadcaster := record.NewBroadcaster()
 	recorder := broadcaster.NewRecorder(scheme, corev1.EventSource{Component: "keda-metrics-adapter"})
-	handler := scaling.NewScaleHandler(mgr.GetClient(), nil, scheme, globalHTTPTimeout, recorder)
+
+	kubeClientset, _ := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
```

Comment:
```suggestion
	kubeClientset, _ := kubernetes.NewForConfig(cfg)
```

Could you please use `cfg` from https://github.com/kedacore/keda/pull/3955 once it is merged?

Reply:
Sure, updated, thanks!